helm-prereqs/cleanup-machine-a-tron.sh

```
@@ -0,0 +1,296 @@
+else
+    ok "release ${RELEASE} not installed"
+fi
+
+# =============================================================================
+# Phase 2 — namespace / cert secret
+# =============================================================================
+phase "Phase 2 — namespace / cert secret"
+if $DELETE_NAMESPACE; then
+    if kubectl get ns "$MAT_NAMESPACE" >/dev/null 2>&1; then
+        confirm "Delete the entire ${MAT_NAMESPACE} namespace (removes pull secret + copied secrets)?" \
+            && { kubectl delete namespace "$MAT_NAMESPACE" --wait=false >/dev/null; ok "namespace ${MAT_NAMESPACE} deletion requested"; } \
```

Shellcheck (0.11.0)

[info] Note that A && B || C is not if-then-else. C may run when A is true.

(SC2015)